test/scripts/crond/test_crontools.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import absolute_import, print_function
+import os, pytest, subprocess
+
+
+__script_dir = os.path.dirname(os.path.abspath(__file__))
+CRONTOOLS_PATH=os.path.realpath(os.path.join(__script_dir, '../../../scripts/crond/crontools.sh'))
+
+def _get_exported_var(script_file, env_var, env=None):
+    script_dir = os.path.dirname(script_file)
+    script_name = os.path.basename(script_file)
+    # subcmd = "cd {0} && source {1} && echo -n ${{{2}}}".format(script_dir, script_name, env_var)
+    subcmd = "source {1} && echo -n ${{{2}}}".format(script_dir, script_file, env_var)
+    # subcmd = ["echo", "$(pwd)"]
+    cmd = ["bash", "-c", subcmd]
+    err = None
+    out = None
+    try:
+      print("ECHO: {0}".format(subprocess.check_output(["/bin/bash", "-c", "echo -n $(pwd)"], env=env, cwd=script_dir, shell=False)))
+      out = subprocess.check_output(cmd, env=env, cwd=script_dir, shell=False)
+      print("OUT: {0}".format(out))
+    except subprocess.CalledProcessError as e:
+      print("ERROR: {0}".format(e.output))
+      err = e.output
+    return (out, err)
+
+@pytest.fixture
+def get_exported_var():
+  return _get_exported_var
+
+@pytest.fixture
+def boilerplate_env(config_file):
+  cur_env = os.environ
+  return cur_env.update({
+    "SIBIS_PYTHON_PATH": os.environ.get("SIBIS_PYTHON_PATH") if os.environ.has_key("SIBIS_PYTHON_PATH") else os.path.realpath(os.path.join(__script_dir, '../../../../')),
```

Comment:
```python
os.environ.get("SIBIS_PYTHON_PATH", os.path.realpath(os.path.join(__script_dir, '../../../../'))),
```
?

Reply:
Oops! I didn't meant to leave that in there... albeit, according to how this is installed, it's technically ends up being in the _right place_ which is the directory above the repo root.